lib/rucio/rse/protocols/protocol.py

```diff
@@ -16,11 +16,10 @@
 This module defines the base class for implementing a transfer protocol,
 along with some of the default methods for LFN2PFN translations.
 """
-
 import hashlib
 import logging
 from configparser import NoOptionError, NoSectionError
-from typing import TypeVar
+from typing import Callable, Tuple, TypeVar
```

Comment:
```suggestion
from typing import Callable, TypeVar
```

built-in `tuple` should be used instead of `Tuple`: https://github.com/rucio/rucio/blob/master/pyproject.toml#L99